docs/References/troubleshooting.md

```diff
@@ -57,3 +57,53 @@ If you are unable to access your cluster via kubectl from the dashboard, check o
   - *Try killing & restarting the prompt pod*
   - *Check logs from Relay Agent to see if there is any error*
   - *SSL certificate mismatch/error, Ensure that you use the correct SSL certificates as the ingress tends to reject incorrect ones*
+
+---
+
+## Frequently Asked Questions
+
+### 1. What is Paralus?
+
+An open source access manager for Kubernetes clusters, Paralus enables teams to provide secure access to Kubernetes clusters. It lets you create and manage access control policies for people, teams and services across multiple Kubernetes clusters without requiring any modifications to your firewall.
+
+Users can simply import K8 clusters into Paralus projects and define rights on a project to project basis. You can also use Paralus to set up any SSO service using GitHub, Azure AD, Okta, etc. so that users can sign-in onto their clusters with the access rights they were given. 
```

Comment:
Have we tried azure ad ? let's add the names of ones we have tried and tested only.